### ISSUE TITLE
Renovate finds package to update but doesn't submit a PR

### ISSUE THREAD

**Issue by sylvestre** [renovatebot/config-help#980]
Trying on Firefox https://github.com/sylvestre/gecko-dev/
in the log, I can see that renovate clearly finds stuff to update in this file: https://github.com/sylvestre/gecko-dev/blob/master/tools/moztreedocs/requirements.in

For example, I can see in the log:
```

          {
            "depName": "Sphinx",
            "currentValue": "==3.2.1",
            "datasource": "pypi",
            "fromVersion": "3.2.1",
            "depIndex": 41,
            "warnings": [],
            "fixedVersion": "3.2.1",
            "sourceUrl": "https://github.com/sphinx-doc/sphinx",
            "homepage": "http://sphinx-doc.org/",
            "updates": [
              {
                "fromVersion": "3.2.1",
                "toVersion": "3.3.1",
                "newValue": "==3.3.1",
                "newMajor": 3,
                "newMinor": 3,
                "updateType": "minor",
                "isSingleVersion": true,
                "isRange": true,
                "releaseTimestamp": "2020-11-12T14:22:13"
              }
            ]
          },
```

I got 19 PR to update version but not more:
https://github.com/sylvestre/gecko-dev/pulls

I do NOT have set prConcurrentLimit.

My config is https://github.com/sylvestre/gecko-dev/blob/master/renovate.json

**Comment by sylvestre** [comment]
Oh, many  thanks.
What about updating the doc? My expectations was that, if I was using config:base, I would get the default behavior.
https://docs.renovatebot.com/configuration-options/#prconcurrentlimit

Maybe updating to add something in the doc stating something like "config:base comes with a default at 20 PR".
I can write the PR if you agree

**Comment by rarkins** [comment]
Renovate **the tool**'s default config is aimed to be as non-opinionated as possible, but the `config:base` preset that the hosted app proposes in onboarding Pull Requests is moderately opinionated. The problem here is that you're reading the docs about the tool's default settings but your repository contains the `config:base` preset. It would be messy and confusing if we tried to annotate the docs with "..unless you include a preset that changes this" clauses on every option.

If you take a look at the onboarding config, it describes what you're seeing:

![image](https://user-images.githubusercontent.com/6311784/99386418-b92e6200-28d2-11eb-9311-bd5e4281139a.png)

In other words, repositories get an onboarding config, and that onboarding config is saying accurately what settings you are accepting by merging it. I think the other two approaches are undesirable in comparison:
- Make the default settings opinionated like `config:base`, or
- Add a warning to dozens of fields in the documentation that.. they can be changed by presets
